pkg/executor/show.go

```diff
@@ -1845,7 +1847,6 @@ func (e *ShowExec) fetchShowCreateUser(ctx context.Context) error {
 		require = privValue.RequireStr()
 	}
 
-	authData := checker.GetEncodedPassword(ctx, e.User.Username, e.User.Hostname)
```

Comment:
I don't quite get why we don't need `ensureActiveUser()` in `fetchShowCreateUser` and other functions. Could you please explain it a little bit, and how can we be sure the correctness can be verified?

Reply:
`show create user` is not `show grants for user`
We just need the command for creating that user, we do not need to assign any privileges to the user. @bb7133